src/main/java/aeneas/models/Model.java

```diff
@@ -91,6 +94,11 @@ public void updateStats() {
       if(getMetadata(activeLevel).getStarsEarned() < stars) {
```

Comment:
That makes it only update the number of stars if the new number is better than the old number. That way if you replay an older level, you don't lose stars.